pybaseball/playerid_lookup.py

```diff
@@ -30,16 +30,20 @@ def chadwick_register(save: bool = False) -> pd.DataFrame:
 
     print('Gathering player lookup table. This may take a moment.')
     s = requests.get(url).content
-    mlb_only_cols = ['key_retro', 'key_bbref', 'key_fangraphs', 'mlb_played_first', 'mlb_played_last']
+    mlb_only_cols = ['key_retro', 'key_bbref',
```

Comment:
there are a lot of formatting changes in this PR that don't relate to the bug. it'd be easier to understand what caused the bug and which changes affect it if there were less formatting changes.

Reply:
Oh, yea. Sorry about that. I had an auto-formatter turned on. I'll update shortly with it turned off.

Reply:
Updated in latest checkin